website/layouts/robots.txt

```diff
@@ -0,0 +1,7 @@
+User-agent: *
+
+Disallow: /v0.4.3-docs/
+
+Allow: /docs/
```

Comment:
Should this also include the homepage?

Reply:
sure

Reply:
i've read https://developers.google.com/search/docs/advanced/robots/robots_txt and don't really understand `allow`. it just seems to say google *may* index it, but it's also seems that everything not explictly disallowed is indexed.